PyTado/interface.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, username, password, http_session=None, debug=False):
         else:
             self.log.setLevel(logging.WARNING)
 
-        self.http = Http(username=username, password=password, http_session=http_session)
```

Comment:
The debug parameter is currently not passed to the Http instance.

Reply:
Maybe add another parameter `debug_http` to not spam logs, if you do not need the http logs?

Reply:
Any updates on this? I mean there is a debug parameter now but it is not working.